### PR TITLE
Gzip transcriptions

### PR DESCRIPTION
**Pull request by emettely**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      
#107 

**Describe what the PR does**    
This is a draft PR for seeing if GZip would work with compression. I've only verified getting a compressed version of transcript would work, without having the payload also.

**State whether the PR is ready for review or whether it needs extra work**    
Potentially more work - if we do make changes in RTE or else where on `paragraphs` and `words`, we would want to recompress and update the Firestore Transcripts collection.

**Additional context**    
Relevant ADRs, branches and investigations
* https://github.com/bbc/digital-paper-edit-firebase/blob/master/docs/ADR/2019-11-27-firestore-saving-timed-transcripts.md#considered-options - in this doc, there's no mention of a conclusion, but Gzip was investigated and was the most favourable option
* Investigation for the compression: https://github.com/bbc/digital-paper-edit-firebase/pull/3 and https://github.com/emettely/compare-json-compression#outcome

tl;dr Gzip had a really good compression rate, saving 88% of the space. This could work well with even 6 hours content (see doc for more deets).

### CODE REVIEW

**Comment by emettely** [comment]
*Where does the compression happen?*
Compression happens at two locations:

- sttChecker - when you get the STT transcription result from PSTT
- in Transcript Editor - when you save the transcription after editing.

*Where does the decompression happen?*
The decompression is done in:

- PaperEditor, when it's loading multiple transcripts in the Paper Editor interface.
